customer.go

```diff
@@ -158,24 +158,28 @@ type CustomerListInput struct {
 }
 
 type CustomerBillingConfigurationInput struct {
-	InvoiceGracePeriod     int                         `json:"invoice_grace_period,omitempty"`
-	PaymentProvider        CustomerPaymentProvider     `json:"payment_provider,omitempty"`
-	PaymentProviderCode    string                      `json:"payment_provider_code,omitempty"`
-	ProviderCustomerID     string                      `json:"provider_customer_id,omitempty"`
-	Sync                   bool                        `json:"sync,omitempty"`
-	SyncWithProvider       bool                        `json:"sync_with_provider,omitempty"`
-	DocumentLocale         string                      `json:"document_locale,omitempty"`
-	ProviderPaymentMethods []ProviderPaymentMethodType `json:"provider_payment_methods,omitempty"`
+	InvoiceGracePeriod                       int                         `json:"invoice_grace_period,omitempty"`
+	SubscriptionInvoiceIssuingDateAnchor     string                      `json:"subscription_invoice_issuing_date_anchor,omitempty"`
+	SubscriptionInvoiceIssuingDateAdjustment string                      `json:"subscription_invoice_issuing_date_adjustment,omitempty"`
```

Comment:
Since they are nullable, shouldn't these be pointers ? Also I believe those should be defined as enums, not string.

Reply:
I believe that if you don't, users won't be able to "unset" the value due to the `omitempty`. You can probably test though.